frontend/src/features/channels/data/schema.ts

```
@@ -279,7 +280,93 @@ export const saveChannelModelPriceInputSchema = z.object({
+  const parsed = z
+    .object({
+      access_token: z.string().min(1),
+      refresh_token: isCopilot ? z.string().optional() : z.string().min(1),
+    })
+    .safeParse(json);
+
+  if (!parsed.success) {
+    ctx.addIssue(issue);
+  }
+}
+function validateOAuthCredentials(
```

Biome (2.4.4)

[error] Shouldn't redeclare 'validateOAuthCredentials'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)